api/src/main/scala/com/gu/adapters/http/AvatarServlet.scala

```diff
@@ -71,7 +71,7 @@ class AvatarServlet(
   }
 
   error {
-    case e: SizeConstraintExceededException =>
+    case _ =>
       RequestEntityTooLarge(
```

Comment:
why is the exception type removed here ?

Reply:
but it potentially changes the behaviour of the code right ? 
It previously only caught a particular exception but now it catches anything but the way it handles the error seems very specific to the "file too large" error